release_notes/ocp-4-19-release-notes.adoc

```diff
@@ -523,13 +523,13 @@ For more information, see xref:../security/tls-security-profiles.adoc#tls-profil
 [id="ocp-release-notes-web-console_{context}"]
 === Web console
 
-[id="ocp-administrator-perspective_{context}"]
-==== Administrator perspective
+Starting with {product-title} 4.19, the perspectives in the web console have unified to simplify navigation, reduce context switching, streamline tasks, and provide users with a more cohesive {product-title} experience.
 
-[id="ocp-developer-perspective_{context}"]
-==== Developer Perspective
+With this unified design, there is no longer a *Developer* perspective in the default view; however, _all_ {product-title} web console features are discoverable to all users. If you are not the cluster owner, you might need to request permission for certain features from the cluster owner. You can still enable the *Developer* perspective.
```

Comment:
```suggestion
With this unified design, there is no longer a *Developer* perspective in the default view; however, _all_ {product-title} web console features are discoverable to all users. If you are not the cluster owner, you might need to request permission for certain features from the cluster owner. The *Developer* perspective can still be manually enabled if you prefer.
```
What do you think about adding a few extra words here for context? There is no longer a dev perspective by default. @JoeAldinger

Reply:
Yep, that's solid. I like it.